pynitrokey/cli/nethsm.py

```diff
@@ -835,6 +849,18 @@ def set_backup_passphrase(
 
     This command requires authentication as a user with the Administrator
     role."""
+
+    print(
```

Comment:
In my opinion this warning should only be printed if the force flag is false. But may this is more a matter of taste.

Reply:
I would be conservative here:  Always printing the warning doesn’t cost us anything, but it could still help to prevent mistakes.